.github/workflows/release-please.yml

```diff
@@ -0,0 +1,24 @@
+on:
+    push:
+      branches:
+        - master
+  
+  permissions:
+    contents: write
+    pull-requests: write
+  
+  name: release-please
+
+  jobs:
+    release-please:
+      runs-on: ubuntu-latest
+      steps:
+        - uses: google-github-actions/release-please-action@v4
```

Comment:
They mention the need to work with a new action:
googleapis/release-please-action@v4
Please change this.

Reply:
Changed